mrp_multi_level/views/mrp_area_views.xml

```diff
@@ -35,7 +35,7 @@
                     <group colspan="4" col="2">
                         <group>
                             <field name="active" invisible="1" />
-                            <field name="warehouse_id" />
+                            <field name="warehouse_id" options="{'no_create': True}" />
                             <field name="location_id" />
```

Comment:
Can you also add the same for location_id?

Reply:
Is it necessary? The “Storage Locations” feature does not affect the creation of new locations, only new warehouses. So it doesn't fail in this field

Reply:
not strictly but to avoid errors, not a good place to create locations on the fly.